addons/wp/functions/fnc_taskPatrol.sqf

```diff
@@ -12,6 +12,7 @@
  * 4: Area the AI Camps in, default [] <ARRAY>
  * 5: Dynamic patrol pattern, default false <BOOL>
  * 6: enable dynamic reinforcement <BOOL>
+ * 7: Teleport group <BOOL>
```

Comment:
This needs far better wording. The [comment](https://github.com/nk3nny/LambsDanger/pull/428/files#diff-a7c7c916574286f9ebceb75bd0d1ec5b509b333c885b6dc9b10c0d8b5ddc5840R123) for the associated code says it teleports the group to a random waypoint position, which itself is confusing cuz it means a random waypoint from the ones the group currently has.